charts/caddy-ingress-controller/values.yaml

```diff
@@ -20,10 +20,10 @@ ingressController:
   rbac:
     create: true
 
-  # If setting autotls the following email value must be set
-  # to an email address that you manage
-  autotls: false
-  email: ""
+  config:
+    acme-ca: https://acme-staging-v02.api.letsencrypt.org/directory
+    email: embraser01@gmail.com
+    debug: "false"
```

Comment:
maybe the values should be bool and than we can format in configmap as string..

Reply:
Ok so I tried it but couldn't transform it in the configmap with Helm. 
I don't really know what could replace this:
`  {{- toYaml .Values.ingressController.config | nindent 2 }}`

Reply:
I'll be more punctual:
```
  acme-ca: {{ .Values.ingressController.config.acme-ca }}
  email: {{ .Values.ingressController.config.email }}
  debug: {{ .Values.ingressController.config.debug | quote }}
```

Reply:
Found a solution to quote every field dynamicaly without having to maintain this field list everywhere (now we only have it in values.yaml, values.schema.yml and in configmap.go